tests/test_document.py

```diff
@@ -16,24 +36,18 @@ def teardown(self):
         self.ctx.pop()
         super(TestDocument, self).teardown()
 
-    def test_new(self):
+    @staticmethod
```

Comment:
Is there a reason why all of these test methods are `@staticmethod`s? It's conventional for test methods in a `unittest.TestCase` to be instance methods.